language_service/src/tests.rs

```diff
@@ -210,18 +210,19 @@ async fn completions_requested_after_document_load() {
 
     worker.apply_pending().await;
 
-    // this should be empty, because the doc hasn't loaded
-    assert_eq!(
-        ls.get_completions(
+    expect![[r#"
+        334
+    "#]]
+    .assert_debug_eq(
+        &ls.get_completions(
             "foo.qs",
             Position {
                 line: 0,
-                column: 76
-            }
+                column: 92,
+            },
         )
         .items
         .len(),
-        13
     );
```

Comment:
Would it aid debugging test failures to compare the lists instead of the number of items in the lists?

Reply:
Not exactly, since the contents of the list don't matter for the purposes of this test. But your comment was the push I needed to change this test into something a bit smarter instead of this opaque number that changes every time we touch something.